src/Http/Controllers/DashboardController.php

```diff
@@ -24,6 +26,101 @@ public function showDashboard()
     }
 
     /**
+     * Shows missing configurations view.
+     *
+     * @return \Illuminate\View\View
+     */
+    public function showConfiguration()
+    {
+        $hasComponents = (Component::withTrashed()->count() > 0);
+        $hasIncidents = (Incident::withTrashed()->count() > 0);
+        $hasCustomized = (Setting::get('style_background_color') || Setting::get('style_text_color'));
+
```

Comment:
@cachethq/owners Not sure about this checks what do you think? Maybe we could have a Setting for each check like `has_customized => true`?